src/utils/calculateReClammInitAmounts.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    type InputAmount,
+    type MinimalToken,
+    type CreatePoolReClammInput,
+    isSameAddress,
+    MathSol,
+} from 'src';
+import { computeInitialBalanceRatio } from '@balancer-labs/balancer-maths';
+
+export async function calculateReClammInitAmounts({
+    tokens,
+    givenAmountIn,
+    createPoolInput,
+}: {
+    tokens: MinimalToken[];
+    givenAmountIn: InputAmount;
+    createPoolInput: CreatePoolReClammInput;
+}): Promise<InputAmount[]> {
+    // sort the user provided tokens
+    const sortedTokens = tokens.sort((a, b) =>
```

Comment:
We normally sort by token `index`. Is there any reason not to do that this time?

Reply:
nah i'll make the change to use the SDK's `getSortedTokens` :+1: